LoopKit/DosingDecisionStore.swift

```diff
@@ -131,6 +131,8 @@ public struct StoredDosingDecision {
     public var carbsOnBoard: CarbValue?
     
     public var predictedGlucose: [PredictedGlucoseValue]?
+
+    public var predictedGlucoseIncludingPendingInsulin: [PredictedGlucoseValue]?
```

Comment:
I believe you'll also need to set this new property when a new dosing decision is stored.

Reply:
Sorry, I see it now in `Loop`.